x/evidence/keeper/infraction.go

```diff
@@ -26,19 +27,6 @@ func (k Keeper) HandleEquivocationEvidence(ctx sdk.Context, evidence *types.Equi
 	logger := k.Logger(ctx)
 	consAddr := evidence.GetConsensusAddress()
 
-	if _, err := k.slashingKeeper.GetPubkey(ctx, consAddr.Bytes()); err != nil {
```

Comment:
why is this being deleted here?

Reply:
Bc there is no need for the validators' public keys on consumer chains.